### PR TITLE
Updated `altLabels` to list

### LINKED ISSUE #30: Switch `altLabel` into a list

### PR DESCRIPTION
**Pull request by sami-m-g**
Resolves #30 .

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# add your model's MetaData object here
+# for 'autogenerate' support
+target_metadata = Base.metadata
+database_url = os_getenv(
+    "DATABASE_URL",
+    config.get_main_option(
+        "sqlalchemy.url",
+        "postgresql+psycopg://user:pass@localhost:5432/dbname",
+    ),
+)
```
Comment (jsvgoncalves) [inline alembic/env.py:30]:
```suggestion
from dds_glossary.database import DATABASE_URL
config.set_main_option("sqlalchemy.url", DATABASE_URL)
```

File: alembic/env.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    and not an Engine, though an Engine is acceptable
+    here as well.  By skipping the Engine creation
+    we don't even need a DBAPI to be available.
+
+    Calls to context.execute() here emit the given string to the
+    script output.
+
+    """
+    context.configure(  # pylint: disable=no-member
+        url=database_url,
```
Comment (jsvgoncalves) [inline alembic/env.py:52]:
```suggestion
        url=config.get_main_option("sqlalchemy.url"),
```

**Review by jsvgoncalves — CHANGES_REQUESTED** [review]
Looking good, would just avoid reading DB env var twice so we can import it and add it to the alembic config directly.

File: alembic.ini [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# set to 'true' to search source files recursively
+# in each "version_locations" directory
+# new in Alembic version 1.10
+# recursive_version_locations = false
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+sqlalchemy.url = postgresql+psycopg://user:password@localhost:5432/testdb
```
[the remark this answers is not in the document]
Comment (sami-m-g) [reply alembic.ini:63]: I kept it mainly as a default value for `os.getenv`. Since we removed that, I don't think we want.

Comment (sami-m-g) [inline alembic.ini:63]:
```suggestion

```

File: alembic.ini [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+# in each "version_locations" directory
+# new in Alembic version 1.10
+# recursive_version_locations = false
+
+# the output encoding used when revision files
+# are written from script.py.mako
+# output_encoding = utf-8
+
+
+
```
Comment (sami-m-g) [inline alembic.ini:64]:
```suggestion
```

**Pull request merged**